source/transactions/tests/retryable-commit-errorLabels.yml

```diff
@@ -0,0 +1,133 @@
+runOn:
+    -
+        minServerVersion: "4.3.x"
+        topology: ["replicaset", "sharded"]
+
+database_name: &database_name "transaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  - description: commitTransaction does not retry error without RetryableWriteError label
+    clientOptions:
+      retryWrites: false
+    failPoint:
+      configureFailPoint: failCommand
+      mode: { times: 1 }
+      data:
+        failCommands: ["commitTransaction"]
+        errorCode: 11600 # InterruptedAtShutdown, normally a retryable error code
+        errorLabels: []
+    operations:
+      - name: startTransaction
+        object: session0
+      - name: insertOne
+        object: collection
+        arguments:
+        session: session0
+        document:
+          _id: 1
+        result:
+        insertedId: 1
+      - name: commitTransaction
+        object: session0
+        result:
+          errorLabelsContain: ["RetryableWriteError"]
```

Comment:
AFAIK, the driver only adds this label for pre-4.4 servers. This test also isn't expecting the commit to be retried, so shouldn't we assert that the error label is omitted?